src/sickle.ts

```diff
@@ -241,6 +246,14 @@ class Annotator {
       this.emit(jsDocTag);
       this.emit(' {');
     }
+    this.visitType(type);
+    if (jsDocTag) {
+      this.emit('}');
+    }
+    this.emit(' */');
+  }
+
+  private visitType(type: ts.TypeNode, optional?: boolean) {
```

Comment:
I renamed the functions to be `maybeEmitJSDocType` and `emitType`.  I prefer "emit" to "visit" because it's clearer that we're going to traverse the node for the purposes of writing something.  I agree the names are pretty confusing.